.github/workflows/win.yml

```
@@ -0,0 +1,94 @@
+name: Windows CI
+
+on: [push, pull_request]
+
+jobs:
+  linux-job:
+    runs-on: ubuntu-latest
+    container: crystallang/crystal:latest
```

```suggestion
    container: crystallang/crystal:0.32.1
```

It's better to keep the exact version used and update them upon release.